### DISCUSSION TITLE
关于xmake教程

### DISCUSSION THREAD

**Discussion by x1244** [xmake-io/xmake#5942] [General]
我购买了蓝桥云课会员准备系统的学习下xmake，但是第一步就卡住了，环境配置不上。
![xmake](https://github.com/user-attachments/assets/74ba0c93-7477-45b1-a56a-99881366880c)

这个好像是libcurl版本冲突导致的，请问怎么解决。
另外，这个学习室的资料是不是有些旧了，请问更新的计划没有？

**Comment by waruqi** [comment]
> 这个好像是libcurl版本冲突导致的，请问怎么解决。

这是蓝桥那个环境问题，得找蓝桥那边的人看下，等下周一把，这个其实跟 xmake 本身没啥关系。

> 另外，这个学习室的资料是不是有些旧了，请问更新的计划没有？

里面的大部分东西都没变，讲的都是基础的入门东西，即使更新，也是出进阶教程，现有基础这块不会有太大改动。。顶多也就是升级下 xmake 版本

**Comment by waruqi** [comment]
这边没有任何问题，而且 git 原本就是环境自带的，不需要额外安装

<img width="654" alt="image" src="https://github.com/user-attachments/assets/310811cb-0bed-49a7-a17c-0ff0475a5961">

**Comment by waruqi** [comment]
这块不执行，暂时用系统自带的吧，应该也可以。这种其实都是蓝桥的实验环境问题，那边会不定期维护更新，可能会跟教程里面有些差异

<img width="411" alt="image" src="https://github.com/user-attachments/assets/3c029b9b-070d-4ace-a58d-28322f8c7b2f">

**Comment by x1244** [comment]
好的，看看资料也是好的，能更系统的掌握一下。

**Comment by x1244** [comment]
今天又遇到了个问题，环境重置，安装xmake，连接不上，这肯定也是环境的问题。
可是我们现在的环境就这样，再试吧。

**Reply by waruqi** [reply]
它们那个环境是这样，一直在变的，或者你可以直接先用自己本地的机器来实验，回头等空了，我教程里面再更新下适配他们的环境调整
